doc/man1/openssl-verification-options.pod

```diff
@@ -83,7 +89,7 @@ if and only if all the following conditions hold:
 
 =item *
 
-It is an an element of the trust store.
+It is an element of the trust store.
```

Comment:
The rules are incomplete. If positive EKUs are specified, at least one must match any EKU associated with the intended purpose.

Reply:
done